unnamed_toaster/unnamed_toaster_hw_interface/src/unnamed_toaster_hw_interface/robot_control.py

```diff
@@ -50,5 +56,7 @@ def callback(self, msg):
         thro = clamp(thro, self.max_speed * -1, self.max_speed)
         steerage = clamp(steerage, self.max_spin * -1, self.max_spin)
 
-        self.table.putNumber("coprocessorPort", thro + steerage) # Set port wheels in m/s
-        self.table.putNumber("coprocessorStarboard", thro - steerage) # Set starboard wheels in m/s
+        self.table.putNumber("coprocessorPort", thro +
+                             steerage)  # Set port wheels in m/s
```

Comment:
this line is a little wonky - suggest comment on preceding line, full putNumber on single line

Reply:
Yeah that is a little weird, changed in the next commit.